### ISSUE TITLE
A question about group_points in op.py

### ISSUE THREAD

**Issue by LJL36** [Blueprintf/pointlstm_gesture_recognition_pytorch#11]
Hi @Blueprintf 
Thanks for sharing your great work!
I have a question about the group_points function in experiments/models/op.py
```python
def group_points(self, distance_dim, array1, array2, knn, dim):
        matrix, a1, a2 = self.array_distance(array1, array2, distance_dim, dim)
        dists, inputs_idx = torch.topk(matrix,
                                       knn,
                                       -1,
                                       largest=False,
                                       sorted=True)
        neighbor = a2.gather(
            -1,
            inputs_idx.unsqueeze(1).expand(dists.shape[:1] + (a2.shape[1], ) +
                                           dists.shape[1:]))
        offsets = array1.unsqueeze(dim + 1) - neighbor
        offsets[:, :3] /= torch.sum(offsets[:, :3]**2,
                                    dim=1).unsqueeze(1)**0.5 + 1e-8
        return offsets
```
The dim for timestamp is also involved in the calculation of the offsets. So the corresponding value is always 0. Is it which you expected?

**Comment by ycmin95** [comment]
@LJL36，
Yes, the timestamp offset in group_points() is always 0 and doesn't affect the group results. As we introduced in the paper (Figure. 3), we extract the intra-frame structure in the stage-1 to reduce the temporal temporal impacts from other frames in the early stage of the feature extraction.

**Comment by LJL36** [comment]
> 
> 
> @LJL36，
> Yes, the timestamp offset in group_points() is always 0 and doesn't affect the group results. As we introduced in the paper (Figure. 3), we extract the intra-frame structure in the stage-1 to reduce the temporal temporal impacts from other frames in the early stage of the feature extraction.

In other words, the dimension of timestamp is redundant because this dimension is all set to zero in stage-1 at the beginning. In fact, the information about the relative position between frames is extracted through the inter-frame stage and the lstm module?

**Comment by ycmin95** [comment]
@LJL36 
Yes, we heuritically desigh the baseline network ( like 2D conv + LSTM) based on previous work (BMVC'19) without detailed ablation studies about the basic module design.

**Comment by LJL36** [comment]
Okay, I see, thanks very much for your patience.
